cob_moveit_bringup/bin/add_robot.sh

```diff
@@ -2,10 +2,10 @@
 
```

Comment:
This file have to be updated @ipa-mdl do you know where can be found the .setup_assistant ?